api/config/config_local.json

```diff
@@ -23,17 +23,25 @@
     "projectNamePrefix": "aou-rw-local1-",
     "exportBigQueryTable": "all-of-us-workbench-test-bd.billing_data.rw_test_firecloud_view",
     "defaultFreeCreditsDollarLimit": 300.0,
+    "defaultInitialCreditDollarLimit": 300.0,
     "freeTierCostAlertThresholds": [
```

Comment:
do we need to keep the old ones and user another PR to delete? If our code switches to use the new config, then we can just delete them now. Same for all other configs